code/index.html

```diff
@@ -1,22 +1,110 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Skaftö Open</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+</head>
+
+<header>
+  <!-- Some kind of logo and navbar -->
+  <img class="banner" src="Skaftoopenmagazine.JPG" alt="Skaftö Open Magazine">
+
+  <nav>
+    <a class="links">
+      <a
+        href="https://www.google.com/search?q=skaft%C3%B6+open&oq=skaft%C3%B6+open+&aqs=chrome.0.69i59l2j35i39j0l3j69i61l2.4334j0j7&sourceid=chrome&ie=UTF-8">News</a>
+      </div>
+      <a
+        href="https://golf.se/for-spelaren/tavlingar/nordic-golf-tour/leaderboard--ranking/#/competition/2258766/leaderboard">Leaderboard</a>
+      </div>
+      <a
+        href="https://golf.se/for-spelaren/tavlingar/nordic-golf-tour/leaderboard--ranking/#/competition/2258766/players">Players</a>
+      </div>
+      <a
+        href="https://golf.se/for-spelaren/tavlingar/nordic-golf-tour/leaderboard--ranking/#/competition/2258766/teetimes">Tee-times</a>
+      </div>
+
+  </nav>
+</header>
+</header>
+
+<!-- The big news content -->
+
+<section class="flex-parent">
+  <div class="flex-child one">
+    <img class="photo-1" src="https://storage.googleapis.com/stateless-svenskgolf-se/2019/12/6480b264-5de36fc64fd0e.jpg"
+      alt="Skaftö Open">
+    <h3>A sunshine story </h3>
+    <p>Learn the story about how the little golf club at the little island manage to found one of the nordics biggest
+      golf events.</p>
+  </div>
+
+  <div class="flex-child two">
+    <video controls loop>
+      <source src="NGT_SKAFTO_V1_1.mp4" type="video/mp4" alt="Your browser does not support the video tag.">
+    </video>
+  </div>
+
+</section>
+
+<!-- The other news - in a grid element  -->
+
+<section class="cards">
+
+  <figure>
```

Comment:
After doing some research, the `<figure>`-tag should be wrapping the `<img>`-tag rather than a `<div>`. Here is some reading if you want to read more about it https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure . In this case I would however remove it completely and just go with the `<div>`. 

BUT, I like that you have been googling and found something by yourself 👏